operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md

```diff
@@ -0,0 +1,185 @@
+# Create a Session Template to Boot Compute Nodes with Scalable Boot Projection Service (SBPS)
+
+When [compute nodes](../../glossary.md#compute-node-cn) are booted, the [Scalable Boot Projection Service (SBPS)](../../glossary.md#scalable-boot-projection-service-sbps)
+projects the root file system \(`rootfs`\) over the network to the compute nodes.ø
+
+This page covers the appropriate contents for a BOS session template in order to use SBPS.
+
+- [Boot set `rootfs_provider` parameter](#boot-set-rootfs_provider-parameter)
+- [Boot set `rootfs_provider_passthrough` parameters](#boot-set-rootfs_provider_passthrough-parameters)
+  - [`<transport>`](#transport)
+  - [`<schema version>`](#schema-version)
+  - [`<IQN Domain>`](#iqn-domain)
+  - [`<DNS SRV record reference>`](#dns-service-srv-record-reference)
+  - [`<client discovery timeout in seconds>`](#client-discovery-timeout-in-seconds)
+  - [`<ramroot>`](#ramroot)
+  - [Example `rootfs_provider_passthrough`](#example-rootfs_provider_passthrough)
+- [`root=` kernel parameter](#root-kernel-parameter)
```

Comment:
I like that you have a section here explaining how the `root` kernel command-line parameter gets constructed by BOS from the contents of the session template, but putting it in this list and at the same level as things like `rootfs_provider` and the "Boot set S3 parameters" is confusing because those are things that the user of BOS has to specify whereas the `root` Kernel command-line parameter gets generated by BOS from these other parameters.

Reply:
I moved this into and appendix that explicitly tells the user no action is required on their part.